### ISSUE TITLE
$ is always interpreted as inline LaTeX

### ISSUE THREAD

**Issue by chreekat** [vim-pandoc/vim-pandoc-syntax#153]
Thus, the line "Here is a line with $40" highlights incorrectly.

I know what a nightmare it is to write these syntax files. This probably worked fine at one time...

I built a test framework for them; perhaps I could polish it up and make it useful to you? Right now it's embedded in my 3%-completed https://github.com/chreekat/vim-haskell-syntax. Helps with finding regressions, which are basically guaranteed when working with vim syntax declarations. We hates them, we does.

**Comment by larryhynes** [comment]
Indeed. I've just run into the same thing.

![screen shot 2015-10-21 at 11 58 46](https://cloud.githubusercontent.com/assets/1852005/10634670/63ee2fa2-77eb-11e5-8050-3da233316693.png)

**Comment by fmoralesc** [comment]
As a workaround, escape the $:

~~~
this $20 works.

this should be highlighted correctly.
~~~

**Comment by larryhynes** [comment]
Yep, that works alright. I had been doing that, but then ran into problems using the same markdown with another renderer. Not the end of the world, though! :-) Thanks for all your work on this plugin.

**Comment by fmoralesc** [comment]
Sorry for not being more helpful, but I haven't really had much time to
work on this :/

**Comment by chreekat** [comment]
For my part, I understand how it goes. :) No worries, good luck.
